plugins/spoiler-alert/assets/javascripts/initializers/spoiler-alert.js

```diff
@@ -28,7 +28,9 @@ export function initializeSpoiler(api) {
   });
 
   addTagDecorateCallback(function () {
-    if (this.element.attributes.class === "spoiled") {
+    const { attributes } = this.element;
+
+    if (/\bspoiled\b/.test(attributes.class)) {
```

Comment:
Ideally we would share the regex but that's fine

Reply:
Yeah, I hesitated and then resorted to the rule of 3 😅